src/main/resources/locales/cy/cookie-policy.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Not specific to this file, however alongside the below we should also ensure that the cookies are disabled when the user declines them.

I believe theres code elseware in this repo that we can plugin to do this.